components/signup/signup.tsx

```diff
@@ -550,6 +571,14 @@ const Signup = ({onCustomizeHeader}: SignupProps) => {
 
     const handleReturnButtonOnClick = () => history.replace('/');
 
+    const handleOnBlur = (e: FocusEvent<HTMLInputElement>, inputId: string) => {
+        const text = e.target.value;
```

Comment:
Could not take the input name/id from the target?

Reply:
it could, but I noticed that for the password input it will look something like password-input , with the dash in the middle, so to keep the logic simple, the ids consistent and easy to modify the id in case we need it, I'd rather to send the inputId as a parameter